apps/api/v2/src/modules/oauth-clients/controllers/oauth-clients/oauth-clients.controller.ts

```diff
@@ -109,6 +116,21 @@ export class OAuthClientsController {
     return { status: SUCCESS_STATUS, data: client };
   }
 
+  @Get("/managed-users/:clientId")
```

Comment:
I think it makes sense to have `@Get("/:clientId/managed-users")` because IMO users can be seen as sub-resources of an OAuth client aka OAuth client is higher up in the hierarchy.